src/streaming.jl

```diff
@@ -215,7 +215,7 @@ function parsefile(filename::AbstractString,callbacks::XPCallbacks; bufferlines=
                 write(io, readline(file))
                 i += 1
             end
-            txt = Compat.bytestring(io)
+            txt = Compat.String(io)
```

Comment:
It does, but it defines the `ByteString` constructor on IOBuffer. I think the problem is to do with the fact `String` is deprecated in favor of `AbstractString` on v0.4.